WordPress/Classes/ViewRelated/NUX/CreateAccountAndBlogViewController.m

```diff
@@ -699,8 +699,6 @@ - (void)createUserAndSite
     WPAsyncBlockOperation *userCreation = [WPAsyncBlockOperation operationWithBlock:^(WPAsyncBlockOperation *operation){
         void (^createUserSuccess)(id) = ^(id responseObject){
             // Turn on the new editor only for users that create a new account within the iOS app
-            [WPPostViewController setNewEditorAvailable:YES];
-            [WPPostViewController setNewEditorEnabled:YES];
             [WPAnalytics track:WPAnalyticsStatEditorEnabledNewVersion];
```

Comment:
> Do we still need to log that the new editor enabled in analytics if it's on by default?

Yup - the goal is to get a feel for people who _toggle_ the editor on/off to compare high level perf stats for legacy vs new